neo/renderer/RenderSystem_init.cpp

```diff
@@ -398,6 +403,15 @@ static void R_CheckPortableExtensions( void ) {
 	if ( glConfig.twoSidedStencilAvailable )
 		qglActiveStencilFaceEXT = (PFNGLACTIVESTENCILFACEEXTPROC)GLimp_ExtensionPointer( "glActiveStencilFaceEXT" );
 
+#if OSX_TIGER || OSX_LEOPARD
+	if( glConfig.glVersion >= 2.0) {
```

Comment:
I'm not sure this works.
If the glversion is >= 2.0, `glStencolOpSeparate` must exist because it's part of OpenGL 2.0 - that ...Ati function is an extension that doesn't have to exist.

Maybe a better solution is to undo all that `PFNGLSTENCILOPSEPARATEATIPROC` code and add something like
```c
#if defined(OSX_TIGER) || defined(OSX_LEOPARD)
typedef void (APIENTRYP PFNGLSTENCILOPSEPARATEPROC) (GLenum face, GLenum sfail, GLenum dpfail, GLenum dppass);
#endif
```

Reply:
I'd have assumed that too but the glext.h files that ship with 10.4 and 10.5 definitely only define glStencilOpSeparateATI, presumably because the only OpenGL2 capable hardware supported by those two platforms is the ATI HD2000 series. That being said, my G5 has an X1900XT which is only OpenGL 1.5 so I'm happy to go with this solution until I'm able to acquire an iMac 7,1 for testing.

Reply:
If you're gonna use glStencilOpSeparateATI, `glversion >= 2.0` is no valid check for that.
You should use something like `if( R_CheckExtension( "GL_ATI_separate_stencil" ) )`

I'd suggest still adding a custom typedef for PFNGLSTENCILOPSEPARATEPROC (for OSX_TIGER and OSX_LEOPARD) and use the ATI-version as a fallback if `glversion < 2.0`

Reply:
if you provide your own `PFNGLSTENCILOPSEPARATEPROC` typedef a nice side-effect is that you can just use that instead of `PFNGLSTENCILOPSEPARATEATIPROC` (the function signatures are identical if I didn't miss anything), which means you won't need most of the changes you did, but only the fallback to loading `"glStencilOpSeparateATI"` in the else-case of `if( glConfig.glVersion >= 2.0)`

Reply:
Sounds good, I'll have a look later this evening :)